ci/tools.go

```diff
@@ -17,7 +17,6 @@ limitations under the License.
 package tools
 
 import (
-	_ "github.com/envoyproxy/protoc-gen-validate"
```

Comment:
Any specific reason for this?

Reply:
hmm i don't know why this got removed, i'll restore it